Serverless/Azure Functions/About Azure Functions.md

```diff
@@ -1,36 +1,57 @@
-Azure Functions is a Function-as-a-Service ("FaaS"), i.e. a platform for running configurable blocks of code that run in response to some event. Functions support several different programming languages such as C# and JavaScript, F#, Java, Python, and PowerShell.
+[Azure Functions](https://learn.microsoft.com/en-us/azure/azure-functions/functions-overview) is Microsoft's serverless solution. They are, unsurprisingly, hosted on Azure, Microsoft's cloud service. Functions support several different programming languages such as C# and JavaScript, F#, Java, Python, and PowerShell.
```

Comment:
Maybe remove `and` between the C# and JavaScript at the end? Just say `... languages such as C#, JavaScript, F#, Java, Python, and Powershell`